src/sentry/api/endpoints/project_details.py

```diff
@@ -61,10 +61,13 @@ def update_project_scenario(runner):
         )
 
 
-def clean_newline_inputs(value):
+def clean_newline_inputs(value, should_lower=True):
     result = []
     for v in value.split('\n'):
-        v = v.lower().strip()
+        if should_lower:
```

Comment:
Reduce this to:

```python
if should_lower:
    v = v.lower()
v = v.strip()
```

It avoids the extra `else` branch and they both ultimately need stripped. The only change is one path is `lower()` ing.

Overall `should_lower` is a weird word to me. Typical nomenclature would be `case_sensitive`/`case_insensitive`. This would also go in line with `re.I`, etc.